intelmq_manager/static/js/static.js

```diff
@@ -525,9 +525,24 @@ $(document).ready(function() {
                 $('#loginErrorField').text("")
                 $('#modalLoginForm').modal('hide');
                 updateLoginStatus();
+            } else if (typeof data.error !== 'undefined') {
+                // If authentication failed, the returned error message is displayed.
+                $('#loginErrorField').text(data.error);
             } else {
-                // If authentication failed, this error message is displayed.
-                $('#loginErrorField').text('Invalid username and/or password');
+                // Other error, display the response for easier debugging.
+                $('#loginErrorField').text("Login failed, server response was " + data);
+            }
+        })
+        .fail(function(jqXHR, textStatus) {
+            if (typeof jqXHR.responseJSON !== 'undefined' && typeof jqXHR.responseJSON.errors !== 'undefined') {
+                let concatenated = "";
+                for (let key in jqXHR.responseJSON.errors) {
+                    console.log(jqXHR.responseJSON.errors)
```

Comment:
Maybe only print the errors once if there are multiple?
```suggestion
                console.log(jqXHR.responseJSON.errors);
                for (let key in jqXHR.responseJSON.errors) {
```

Reply:
oops, that's a debugging leftover. but using it for the unhandled error response might be a good idea